### PR TITLE
add template for issues

### PR DESCRIPTION
**Pull request by tooomm**
Simple first working version with the most basic information.
Announcement: https://github.com/blog/2111-issue-and-pull-request-templates

When people use it properly it should look like this:
<pre>
<b>OS:</b>
OS X 10.8.5

<b>Cockatrice version:</b>
2d53ce9

___

Hello, I have a problem. (...)
</pre>
<br>And come out like this:
template

<br>
I tried to make it as user friendly as possible with some space between the lines for easy selecting and pasting, examples etc.<br>
After merging someone else should create a `.github/` folder in the root directory and move the `CONTRIBUTING.md` and `ISSUE_TEMPLATE.md` files there.
I wasn't able to create a folder with a starting dot in windows. :)

### CODE REVIEW

**Comment by Daenyth** [comment]
> I wasn't able to create a folder with a starting dot in windows

Look into git-bash on windows, should help you out.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<b>OS:</b>
+*Override this line with the exact operating system you are running! (e.g. "Win 7 SP1", "OS X 10.8.5", "Ubuntu 15.10" ...)*
+
+<b>Cockatrice version:</b>
+*Put your Cockatrice version number here! You can find it inside the app: `Help` --> `About Cockatrice` (e.g. "2d53ce9")*
```
Comment (Daenyth) [inline ISSUE_TEMPLATE.md]: We should prompt them to include the date.

Comment (ZeldaZach) [inline ISSUE_TEMPLATE.md]: I agree w/ the date. It's one string, a simple c/p

Comment (tooomm) [inline ISSUE_TEMPLATE.md]: Good point! :+1: 

Well, still not possible to copy text there on windows, see https://github.com/Cockatrice/Cockatrice/issues/1762

**Comment by tooomm** [comment]
Windows Explorer is capable of doing this as well. `.github.` did the trick. :+1: 

Any suggestions for a pr template?

**Comment by Daenyth** [comment]
Let's do that in another PR. I'll think about it

**Pull request merged**